Slambook/index.js

```diff
@@ -67,9 +67,32 @@ form.addEventListener('submit',function(e){
     allSlamArray.push(formValue)
     console.log(formValue)
 console.log(allSlamArray)
+showSlamList();
+})
```

Comment:
Variable name 'allSlamArray' is not defined, consider renaming it to match the correct variable name or ensure it is properly defined.